python/ray/experimental/dag/class_node.py

```diff
@@ -133,6 +133,15 @@ def bind(self, *args, **kwargs):
         self._actor._last_call = node
         return node
 
+    def remote(self, *args, **kwargs):
+        raise AttributeError(
+            "All DAGNode types act as intermediate representation (IR) of a "
+            "DAG that can only be produced by .bind() on supported class or "
+            "function, and executed via dag.execute(input). Please do not call "
```

Comment:
Please change this to: ".remote() cannot be used on ClassMethodNodes. Use .bind() instead to express an symbolic actor call."